bloom-housing/bloom
# Alert Banner for Login Issues

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description
Alert Banner was moved to the shared-helpers so it can be used in both partner and public sites. With the maintenance window env var set in both partner and public, the alert banner will appear. Translations created through Google Translate.

## How Can This Be Tested/Reviewed?

Test by setting MAINTENANCE_WINDOW var in partner and public sites. Banner should appear. Check translations work. 
Set a past window or no window, banner should not appear.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
